app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -111,15 +111,15 @@ def provided_attributes
     end
 
     def max_cpu_count
-      16
+      288
     end
 
     def max_socket_count
       16
     end
 
     def max_memory
-      16.gigabytes
+      4096.gigabytes
```

Comment:
Are we able to make the value architecture-wise?
As I understand it, on x86, we can not exceed this value.
Could we at least add an info on the field, to propagate the information to the user:
https://github.com/theforeman/foreman/blob/280aca00634cb30cc88e07a7784fe8e948eeda68/app/views/compute_resources_vms/form/ovirt/_base.html.erb#L34

Reply:
The Virtual Machine pane is before the operating system pane so if the user in in the blank following the order, we don't know with OS will be used (x86 or x86_64 or any other arch). The oVirt OS profile to apply is deduced of OS provided by the user in foreman.

Reply:
Yeah and we still do not have 100% information about the architecture to be used, even though we would hardcode it, because users can change that, in ovirt's configuration right?